docs/devel/gubernator.md

```diff
@@ -0,0 +1,158 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Gubernator
+
+*This document is oriented at developers who want to use Gubernator to debug while developing for Kubernetes.*
+
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [Gubernator](#gubernator)
+  - [What is Gubernator?](#what-is-gubernator)
+  - [Gubernator Features](#gubernator-features)
+    - [Test Failures list](#test-failures-list)
+    - [Log Filtering](#log-filtering)
+    - [Gubernator for Local Tests](#gubernator-for-local-tests)
+  - [Future Work](#future-work)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+## What is Gubernator?
+
+[Gubernator](https://k8s-gubernator.appspot.com/) is a webpage for viewing and filtering Kubernetes
+ test results. Gubernator runs on Google App Engine and fetches logs stored on Google Cloud Storage.
+
+Gubernator simplifies the debugging proccess and makes it easier to track down failures by automating many
+steps commonly taken in searching through logs, and by offering tools to filter through logs to find relevant lines.
+Gubernator automates the steps of finding the failed tests, displaying relevant logs, and determining the
+failed pods and the corresponing pod UID, namespace, and container ID.
+It also allows for filtering of the log files to display relevant lines based on selected keywords, and
+allows for multiple logs to be woven together by timestamp.
+
+## Gubernator Features
+
+### Test Failures list
+
+Issues made by k8s-merge-robot will post a link to a page listing the failed tests.
+Each failed test comes with the corresponding error log from a junit file and a link
+to filter logs for that test.
+
+Based on the message logged in the junit file, the pod name may be displayed.
+
+![alt text](https://github.com/gubernator-images/testfailures.png)
```

Comment:
I think when I run the update-munge-docs.sh script it does that, so I'll put back the full path instead of the relative one.